lib/comply/whitelist_constantize.rb

```diff
@@ -0,0 +1,30 @@
+require 'active_model/validations'
+
+module Comply
+  module WhitelistConstantize
+    class NotWhitelistedError < ArgumentError
+    end
+
+    DEFAULT_PROC = -> (klass) { klass <= ActiveModel::Validations }
+
+    class << self
+      def constantize(str)
+        whitelist! { str.classify.constantize }
```

Comment:
Why not just call this method with an argument? The block/yield isn't necessary.